### DISCUSSION TITLE
Tiger Automatic 1111 fork vs Node-AI

### DISCUSSION THREAD

**Discussion by GoZippy** [nod-ai/SHARK#1418] [General]
Anyone care to chime in on the AMD side of things? I have been following many projects for AMD devices but mostly Node-ai and https://github.com/lshqqytiger/stable-diffusion-webui-directml version of Automatic1111 WebUI for directml.

Node-ai has been a lot more performant for me with my AMD cards but tiger's fork of Auto1111 is benefiting a lot from all the work on the main Auto1111 master with all the plugins and extensions. Image2Image works for me on this but not on Node-ai with RDNA2 device... well it did for a while but does not now. I installed text2video and many other controlnet plugins and can run them OK in CPU only... still a work in progress... 

Just wanted to know if there are any roadmap differences to discuss or if we might be able to benefit from all those other integrations they already worked out in their community we might benefit from here... 

Ideas?

**Comment by Lumpsack** [comment]
Check out VLAD (now SD.next) - I'm pretty sure Tiger moved over to that dues to A1111 moving too slowly - also I know that come ROCM on Windows it'll be added.
